pom.xml

```diff
@@ -283,6 +283,18 @@
         <version>${powermock.version}</version>
         <scope>test</scope>
       </dependency>
+
```

Comment:
do we need this in the top-level package if the annotation is only used in the servers module?